linter.py

```diff
@@ -167,6 +228,30 @@ def _format_message(self, flow_message):
                     snippet = '{} ({})'.format(error_string, error_descr)
             return snippet.strip()
 
+    def _uncovered_to_tuple(self, uncovered, uncovered_lines):
+        """
+        Map an array of flow coverage locations to a fake regex match tuple.
+
+        Since flow produces JSON output, there is no need to match error
+        messages against regular expressions.
+        """
+        match = self.filename == uncovered.get('source')
+        line = uncovered['start']['line'] - 1
+        col = uncovered['start']['column'] - 1
+        is_error = False
+        is_warning = True
+
+        # SublimeLinter only uses the length of `near` if we provide the column
+        # That's why we can get away with a string of the right length.
+        near = ' ' * (uncovered['end']['offset'] - uncovered['start']['offset'])
+
+        message = '\u3003' # ditto mark
+        if line not in uncovered_lines:
+            message = 'Code is uncovered by Flow'
+            uncovered_lines.add(line)
```

Comment:
This might not be the best idea in the world, but it prevents multiple occurrences of “Code is not covered by flow” in a single line (Flow may report multiple areas per line).